app/controllers/score_controller.py

```diff
@@ -0,0 +1,170 @@
+from app.models.user import User
+from app.models.score import Score
+from app.models.exam import Exam
+from app import response, app, db
+from flask import request, jsonify
+import datetime
+from flask_jwt_extended import *
+
+
+def create_score():
+    try:
+        # id = request.form.get('id')
+        user_id = request.form.get('user_id')
+        exam_id = request.form.get('exam_id')
+        score = request.form.get('score')
+
+        new_score = Score(
```

Comment:
I don't see an `__init__` method in your `Score model`, will this work?

Reply:
I have tried it and tested it, and it works. Please tell me if you found it not working. I will immediately check it.